src/ZODB/interfaces.py

```diff
@@ -280,9 +281,9 @@ class IStorageWrapper(Interface):
     - Out-of-band invalidation support
 
       A storage can notify it's wrapper of object invalidations that
-      don't occur due to direct operations on the storage.  Currently
-      this is only used by ZEO client storages to pass invalidation
-      messages sent from a server.
+      don't occur due to direct operations on the storage. This is
+      used by non-IMVCCStorage storages like ZEO and NEO to pass
```

Comment:
`storages` should be `client storages`

Reply:
Thanks for pointing this out. I reworked it a bit differently as _"This is used by **client part** of non-IMVCCStorage storages like ZEO and NEO to pass invalidation messages sent from **a** storage server."_

The reason for this language is that in ZODB speak IStorage already represents a client, not storage server. I hope it is ok.